CONTRIBUTING.md

```diff
@@ -37,10 +37,11 @@ If you know Markdown, you can already contribute! How to contribute can be found
 Changes to the project are proposed through pull requests. The general pull request workflow is as follows:
 
 1. Fork the repo and create a topic branch off of `main`.
-2. If you have added code that should be tested, add unit tests.
-3. If you have changed APIs, update the documentation. Make sure the documentation builds.
-4. Ensure all tests and lints pass on each and every commit that is part of your pull request. `cargo fmt && cargo xclippy`
-5. Submit your pull request.
+2. Locally run `./scripts/dev_setup.sh` to ensure you have all development dependencies required for our workflows.
+3. If you have added code that should be tested, add unit tests.
+4. If you have changed APIs, update the documentation. Make sure the documentation builds.
```

Comment:
Note for later, we need instructions on the goldenfile tests for both Move and API